contract_manager/src/chains.ts

```diff
@@ -77,6 +113,40 @@ export class SuiChain extends Chain {
   getType(): string {
     return SuiChain.type;
   }
+
+  private wrapWithWormholeGovernancePayload(
+    actionVariant: number,
+    payload: Buffer
+  ): Buffer {
+    const builder = new BufferBuilder();
+    builder.addBuffer(
+      Buffer.from(
+        "0000000000000000000000000000000000000000000000000000000000000001",
+        "hex"
+      )
+    );
+    builder.addUint8(actionVariant);
+    builder.addUint16(CHAINS["sui"]); // should always be sui (21) no matter devnet or testnet
+    builder.addBuffer(payload);
+    return builder.build();
+  }
```

Comment:
This is because how the governance messages in Sui were implemented. We are heavily using the governance packages of wormhole and therefore we had to conform to their format on top of our own :-\